readthedocs/builds/tasks.py

```diff
@@ -333,5 +333,3 @@ def sync_versions_task(project_pk, tags_data, branches_data, **kwargs):
             promoted_version.active = True
             promoted_version.save()
             trigger_build(project=project, version=promoted_version)
-
-    return list(added_versions), list(deleted_versions)
```

Comment:
We need to update the docstring here. Shoudl we return `True` if it worked, or similar?